components/Ink/src/private/MDCLegacyInkLayer.h

```diff
@@ -0,0 +1,102 @@
+/*
+ Copyright 2015-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
2017 since this is new.

Reply:
Legacy is the old version, written in 2015. Use 2017 if the file is renamed?

Reply:
Done.